File: pdlearning/src/microservices/form-microservice/Api/Application/Models/FormQuestionAnswerStatisticsModel.cs
using System;
using Microservice.Form.Domain.Entities;

namespace Microservice.Form.Application.Models
{
    public class FormQuestionAnswerStatisticsModel
    {
        public FormQuestionAnswerStatisticsModel()
        {
            // Do not remove this constructor. It is being used by JsonSerializer.Deserialize
        }

        public int AnswerCode { get; set; }

        public string AnswerValue { get; set; }

        public double AnswerCount { get; set; }

        public double AnswerPercentage { get; set; }
    }
}

File: pdlearning/src/web-apps/src/libraries/domain-api/src/lna-form/models/form-question-answer-statistics.model.ts
export interface ISurveyQuestionAnswerStatisticsModel {
  answerCode: number;
  answerValue: string;
  answerCount: number;
  answerPercentage: number;
}

export class SurveyQuestionAnswerStatisticsModel implements ISurveyQuestionAnswerStatisticsModel {
  public answerCode: number;
  public answerValue: string;
  public answerCount: number;
  public answerPercentage: number;
}
